webui/src/components/Appliance/Appliances.vue

```diff
@@ -2017,13 +2017,27 @@ export default {
     const selectedBladeStatus = computed(() => bladeStore.selectedBladeStatus);
 
     const statusColor = computed(() => {
-      return selectedBladeStatus.value === "online" ? "#6ebe4a" : "#ff9f40";
+      if (selectedBladeStatus.value === "online") {
+        return "#6ebe4a";
+      } else if (selectedBladeStatus.value === "unavailable") {
+        return "#ff9f40";
+      } else if (selectedBladeStatus.value === "offline") {
+        return "#b00020";
+      } else {
+        return "#B0B0B0"; // Default unknown status
```

Comment:
How does the "unknown" state present itself in the webui?
Can you add an example to the PR description?

Reply:
No, I didn't set the color for the unknown devices, I just mark the default status to the gray color, means if it is not online, offline or unavailable, it will be regraded as default unknown.